src/test/resources/config/empty-elevated-accounts-test-config.yaml

```diff
@@ -43,7 +43,8 @@ rateLimiter:
 redis:
   endpoint: localhost:6379
   ssl: false
-  timeout: 100ms
+  commandTimeout: 100ms
```

Comment:
Is there a reason the test config values are different to the production config defaults?

Reply:
No idea. I've changed them to match the prod config defaults.